config/application.rb

```diff
@@ -121,7 +121,9 @@ def preview?
 
     # @return [Boolean]
     def debug?
-      Rails.env.development? || review?
+      return ENV['DEBUG'] == 'true' if ENV['DEBUG'].present?
```

Comment:
If we'd rather opt-in to debugging in review we should instead make `DEBUG` a docker compose env in the dev setup, add it to `.env.example` for visibility and also copy the dry-types approach.